lib/bundler/dsl.rb

```diff
@@ -107,7 +107,8 @@ def gem(name, *args)
           unless deleted_dep
             return if dep.type == :development
             raise GemfileError, "You cannot specify the same gem twice with different version requirements.\n" \
-                            "You specified: #{current.name} (#{current.requirement}) and #{dep.name} (#{dep.requirement})"
+                            "You specified: #{current.name} (#{current.requirement}) and #{dep.name} (#{dep.requirement}). " \
+                            "If you want to update the gem version, run `bundle update #{current.name}`. You may need to change the version requirement specified in the Gemfile if it's too restrictive"
```

Comment:
Missing a period at the end 🙂

Reply:
A period is prepended and appended to the error message [here](https://github.com/bundler/bundler/blob/master/lib/bundler/dsl.rb#L554)